build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -eu
+
+echo λλλ get newest PHP CS Fixer version
+php download/php-cs-fixer-v2.phar self-update || curl --silent --fail -L "https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/$(curl --silent --fail -u keradus:5e7538aa415005c606ea68de2bbbade0409b4b8c https://api.github.com/repos/FriendsOfPHP/PHP-CS-Fixer/releases/latest | jq -r .tag_name)/php-cs-fixer.phar" -o download/php-cs-fixer-v2.phar
```

Comment:
here, we are downloading the stable release (by tags)

as we managed to release PHP CS Fixer v2.15.10, but not v2.16.8+ (due to irrelevant CI issues), i suggest to comment that line if you want to play with the PR - i generated the page using 2.15.10, as we changed how we cross-reference the documents and it got released on 2.15 line, but not yet 2.16.